### DISCUSSION TITLE
Emoji extension issue with heart rendering

### DISCUSSION THREAD

**Discussion by Adam-Collier** [ueberdosis/tiptap#4747] [General Questions]
Hi guys, 

I came across an issue today using the emoji plugin which I hope you might be able to help with (or point me in the right direction). When trying to use the heart emoji (it could be others too) it doesn't render as expected:
<img width="217" alt="image" src="https://github.com/ueberdosis/tiptap/assets/20065745/d0e88262-573c-424b-b5b4-7f35d0d85c9c">

However, when using it in the usage section of the emoji extension page it works as expected! 
<img width="307" alt="image" src="https://github.com/ueberdosis/tiptap/assets/20065745/520afe1f-ee5c-4cf8-9187-18e5b483c46d">

Is it an issue with the current version? Could you share what version is currently being used for the usage section?

All the best,
Adam

**Comment by vicpagano** [comment]
Hi! Can you solve this issue @Adam-Collier? I'm facing the same! Thanks in advance!
